### PR TITLE
PowerShell content support

### PR DESCRIPTION
**Pull request by Itay4**
## Status
Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/17220

## Related PRs

description | PR
------ | ------
previous content pr| https://github.com/demisto/content/pull/3691
server | https://github.com/demisto/server/pull/13566

## Required version of Demisto
5.5

## Does it break backward compatibility?
   - No

### CODE REVIEW

File: Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1 [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+.PARAMATER removeNull
+
+Remove empty columns from the table.
+
+.PARAMATER metadata
+
+Metadata about the table contents
+#>
+
+    Write-Host 'tableToMarkdown'
```
[the remark this answers is not in the document]
Comment (Itay4) [reply Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1]: master pwsh (aka @yardensade ) might be able to answer that

[the remark this answers is not in the document]
Comment (yardensade) [reply Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1]: Will take few weeks to engineer that, there are no existing solutions out there atm.
We should however put the the following line in start of file:
`$progressPreference = 'silentlyContinue'`

Comment (yardensade) [inline Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1]:
```suggestion
```

File: Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1 [hunk 2]
```diff
@@ -0,0 +1,34 @@
+function tableToMarkdown {
```
Comment (yardensade) [inline Scripts/CommonServerPowerShell/CommonServerPowerShell.ps1]:
```suggestion
# Silence Progress STDOUT (e.g. long http request download progress)
$progressPreference = 'silentlyContinue'
function tableToMarkdown {
```

**Comment by yaakovi** [comment]
@Itay4 @yardensade 
what is the status here?
waiting for the tableToMarkdown implementation?

**Comment by Itay4** [comment]
pending the related PRs

**Pull request merged**